src/filters/quantitave-filter.js

```diff
@@ -0,0 +1,130 @@
+import React, { useMemo, useState } from 'react'
+import PropTypes from 'prop-types'
+
+import Slider from '@material-ui/core/Slider'
+import { makeStyles } from '@material-ui/core/styles'
+import TextField from '@material-ui/core/TextField'
+import { Button } from '@material-ui/core'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '40ch',
+    padding: theme.spacing(4, 2, 0, 2),
+    fontSize: theme.typography.fontSize,
+    textAlign: 'center',
+  },
+
+  rangeContainer: {
+    display: 'grid',
+    'grid-template-columns': '1fr 1fr',
+    columnGap: 10,
+    textAlign: 'left',
+    textIndent: 5,
+  },
+
+  buttonContainer: {
+    marginTop: 20,
+    display: 'grid',
+    'grid-template-columns': '1fr 1fr',
+    columnGap: 10,
+  },
+}))
+
+const QuantitaveFilter = ({ column: { filterValue, preFilteredRows, setFilter, id, percentage }, closePopper }) => {
+  const [minValue, setMinValue] = useState('')
+  const [maxValue, setMaxValue] = useState('')
+
+  const classes = useStyles()
+
+  const [min, max] = useMemo(() => {
+    let min = preFilteredRows.length ? preFilteredRows[0].values[id] : 0
+    let max = preFilteredRows.length ? preFilteredRows[0].values[id] : 0
+    preFilteredRows.forEach(row => {
+      min = Math.min(row.values[id], min)
+      max = Math.max(row.values[id], max)
+    })
```

Comment:
`Math.min/max` has a built-in reducer so the logic may be more elegantly expressed as:

```js
const values = preFilteredRows.map(({ values }) => values[id])
const min = Math.min(...values, 0)
const max = Math.max(...values, 0)
```